historicState not available on NumberItem anymore in DSL rules

Since a recent snapshot update, a couple of my DSL rules stopped working with the following message:
```
19:58:01.006 [ERROR] [.internal.handler.ScriptActionHandler] - Script execution of rule with UID 'rule-1' failed: 'historicState' is not a member of 'org.openhab.core.library.items.NumberItem'; line 6, column 62, length 49 in rule
```
The according code in the rule is 
```
MyItem.historicState(now.minusMinutes(1))
```
I couldn't yet figure out, which recent change might have introduced this regression, I still have to further investigate.

@J-N-K Do you maybe have an idea as you have a good overview of the recent changes?

--- a/bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/scoping/ScriptImplicitlyImportedTypes.java
+++ b/bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/scoping/ScriptImplicitlyImportedTypes.java
@@ -84,6 +84,8 @@ public class ScriptImplicitlyImportedTypes extends ImplicitlyImportedFeatures {
         result.add(Transformation.class);
         result.add(ScriptExecution.class);
         result.add(URLEncoder.class);
+
+        result.addAll(getActionClasses());
         return result;
     }
 

